ui/pages/confirmations/components/confirm/info/typed-sign/permit-simulation/decoded-simulation/decoded-simulation.tsx

```diff
@@ -1,7 +1,95 @@
-import React from 'react';
+import React, { ReactNode } from 'react';
+import {
+  DecodingDataChangeType,
+  DecodingDataStateChange,
+} from '@metamask/signature-controller';
+import { Hex } from '@metamask/utils';
 
-const DecodedSimulation: React.FC<object> = () => (
-  <div>"DECODED SIMULATION IMPLEMENTATION"</div>
-);
+import { TokenStandard } from '../../../../../../../../../shared/constants/transaction';
+import {
+  Box,
+  Text,
+} from '../../../../../../../../components/component-library';
+import {
+  TextColor,
+  TextVariant,
+} from '../../../../../../../../helpers/constants/design-system';
+import { useI18nContext } from '../../../../../../../../hooks/useI18nContext';
+import { SignatureRequestType } from '../../../../../../types/confirm';
+import { useConfirmContext } from '../../../../../../context/confirm';
+import StaticSimulation from '../../../shared/static-simulation/static-simulation';
+import PermitSimulationValueDisplay from '../value-display/value-display';
+
+const getStateChangeLabelMap = (
```

Comment:
Minor, if this is static, is it simpler to define in a constant using just the translation keys, then call `t` once on use`?

Reply:
Calling `t` for strings is useful to find when strings are not used in translation.